### PR TITLE
Style fix

### PR DESCRIPTION
**Pull request by TuxCoder**
small style fix, remove leading spaces and tabs,

done with this script:
```
find -name '*.xml' -o -name '*.java' -print0 | xargs -0 sed 's#\t$##g' -i
find -name '*.xml' -o -name '*.java' -print0 | xargs -0 sed 's# $##g' -i
```
and run a few times.

edit:
add xml to list

### CODE REVIEW

File: src/main/java/at/roteskreuz/covidapp/config/SecurityConfig.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ protected void configure(HttpSecurity http) throws Exception {
 				.csrf().disable()
 				.authorizeRequests()
 				.antMatchers("/api/v" + appVersion + "/*").permitAll()
-				//.antMatchers("/swagger*").permitAll()
+				.antMatchers("/swagger*").permitAll()
```
Comment (DominicLuidold) [inline src/main/java/at/roteskreuz/covidapp/config/SecurityConfig.java]: Was the comment intentionally removed or was this an unwanted side effect of the style fix?

**Review by DominicLuidold — COMMENTED** [review]
Please have a look at the comment regarding the removed `//` comment in the code that may have unintended side effects.

File: src/main/java/at/roteskreuz/covidapp/config/SecurityConfig.java (hunk 1 above)
Comment (TuxCoder) [reply src/main/java/at/roteskreuz/covidapp/config/SecurityConfig.java]: oh thanks, oversee that, tested some other stuff and did commits between that....

**Comment by DominicLuidold** [comment]
@peterjaschke Is there a specific reason why this pull request was closed without any comment or having stated what's wrong?

**Comment by TuxCoder** [comment]
@DominicLuidold 
See reason: https://github.com/austrianredcross/RCA-CoronaApp-Backend/issues/8

**Pull request closed without merging**